src/CMakeLists.txt~

```diff
@@ -0,0 +1,128 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
I mean that it's like a backup copy created by a text editor (usually this files are used to recover the changes when you don't close the editor properly). Normally this copies are deleted when one closes the editor properly, but in this case it seems it did not do that.

Hint: look at the extension it *.txt~ instead of being *.txt